packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -967,6 +967,15 @@ export class Combobox
     if (!this.allowCustomValues) {
       this.setMaxScrollerHeight();
     }
+
+    this.groupItems.forEach((groupItem, index, array) => {
+      if (groupItem.children.length === 0) {
+        if (index + 1 < array.length) {
+          const nextGroupItem = array[index + 1];
+          nextGroupItem.classList.add(CSS.groupThatFollowsChildlessGroup);
```

Comment:
Can you add a reflected, internal property (`afterEmptyGroup`?) to `combobox-group`s that `combobox` sets here instead of applying the class here?